.github/CONTRIBUTING.md

```diff
@@ -49,7 +53,7 @@ on your local machine to check that building the app still works.
 
 Your PR will be picked up and reviewed before being merged into Pattern Projector proper.
 
-# Doing translations
+### Doing translations
 
 - Set up the development environment as described above.
```

Comment:
The Doing translations section is out of date, sorry! The translations are done using Weblate. Here's the link: https://hosted.weblate.org/projects/pattern-projector/pattern-projector